cmd/ore/aws/copy-image.go

```diff
@@ -53,7 +63,49 @@ func runCopyImage(cmd *cobra.Command, args []string) error {
 		os.Exit(1)
 	}
 
-	err = json.NewEncoder(os.Stdout).Encode(amis)
+	out := make(map[string]CopiedImage)
+	for region, ami := range amis {
+		var snapshotID string
+		err = util.RetryConditional(12, 10*time.Second, func(err error) bool {
```

Comment:
Hmm, 12 seems a bit high. Should we only be retrying on certain error conditions?
If there's context around this, let's throw a link to that if so.

Reply:
Ah yeah, this was originally 6 times at 10 seconds in between and I bumped it while trying to debug what ended up being user error. I'll knock it back down.